src/client/routes/dashboard/index.tsx

```diff
@@ -1,44 +1,42 @@
 import * as React from 'react';
-import { Wrapper } from 'components/Wrapper';
 import { Button } from 'components/Button';
-
-interface IState {
-  counter: number;
-}
+import { bindActionCreators } from 'redux';
+import { connect, Dispatch } from 'react-redux';
+import { actions } from './reducer';
+import WithLoader from 'components/WithLoader';
+import DashboardInfo from './components/DashboardInfo';
+const DashboardInfoWrapper = WithLoader(DashboardInfo);
 
 interface IProps {
```

Comment:
this should extend reduce state interface

Reply:
Done.